src/embed_tests/Python.EmbeddingTest.15.csproj

```diff
@@ -23,7 +23,7 @@
     <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\..\</SolutionDir>
     <PythonBuildDir Condition="'$(TargetFramework)'=='net40' AND '$(PythonBuildDir)' == ''">$(SolutionDir)\bin\</PythonBuildDir>
     <PublishDir Condition="'$(TargetFramework)'!='net40'">$(OutputPath)\$(TargetFramework)_publish</PublishDir>
-    <LangVersion>6</LangVersion>
+    <LangVersion>7.3</LangVersion>
```

Comment:
Is this intentional? @lostmsu

Reply:
Yes, I think I am using a few features. Is there a C# compiler, that does not support 7.3 yet?

Reply:
@lostmsu just wanted to make sure it wasn't added by mistake.